pom.xml

```diff
@@ -65,6 +65,11 @@
       <artifactId>minimal-json</artifactId>
       <version>0.9.5</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
```

Comment:
How come we need two JSON libraries? It would probably be better if we could use just the one.

Reply:
The constructor of `JsonHttpContent` accepts a json key/value which is the JSON body as a map
Gson reliably converts the json body string as a map which minimal-json doesn't not provide an implementation for.

We can write our own, but if Gson provides a concrete implementation

Happy to discuss

Reply:
Having just tried it the minimal-json appears to do a decent job:

`Json.parse(jsonBody).asObject();`

Looking at the source it doesn't look like the JsonHttpContent actually requires a key/value map, simply an object it can serialise back to JSON. Have you tried passing in the `JsonObject` that the above outputs?

If not, if you could be so kind to try using a single stream command to convert the `JsonObject` into a map that would be appreciated, see `names()`.

Reply:
Yup, I've tried with strings and JSON objects as well as a JSON string
The javadoc states `data JSON key name/value data`
Will try converting the JSON object without GSON